sketch-on-map/README.md

```diff
@@ -17,11 +17,12 @@ Use the buttons at the top of the screen to: undo or redo change made to the ske
 ## How it works
 
 1. Create a `GeometryEditor` and pass it to the MapView with `mapView.geometryEditor = geometryEditor`.
-2. Use `GeometryEditor.start(GeometryEditorCreationMode)` to start sketching.
-3. Check to see if undo and redo are possible during a sketch session using `GeometryEditor.canUndo` and `GeometryEditor.canRedo`. If it's possible, use `GeometryEditor.undo()` and `GeometryEditor.redo()`.
-4. Check if sketch is valid using `GeometryBuilder.builder(sketchGeometry).isSketchValid`, then allow the sketch to be saved to a `GraphicsOverlay`.
-5. Get the geometry of the sketch using `geometryEditor.geometry.value`, and create a new `Graphic` from that geometry. Add the graphic to the graphics overlay.
-6. To exit the sketch editor, use `GeometryEditor.stop()`.
+2. Set the `GeometryEditor.tool` to `VertexTool()` for vertex geometry or `FreehandTool()` for free hand geometry
```

Comment:
I see the common samples readme is still referencing the SketchEditor API. I think it would be better if we update it to use the new GeometryEditor API instead.